pkg/networkservice/common/interpose/server.go

```diff
@@ -108,8 +108,6 @@ func (l *interposeServer) Request(ctx context.Context, request *networkservice.N
 			return result, nil
 		}
 
-		l.activeConnection.Delete(activeConnID)
```

Comment:
Why do we need to delete this?

Reply:
When heal fails its first Request, it cleans up active connection - it results in Close failing with `no active connection found` because for `interpose` it looks like trying to close not existing connection. `heal` stands in chain after `interpose`, so client cannot stop healing in such case.

Reply:
Hm... As I can see with this fix activeConnection map will not delete connections. 
Am I get it correctly that we got a leak of active connections?

Reply:
You are right, missed this.
Fixed in other way with setting `clientURL = nil` an invoking `next.Close` in Close in case of no active connection exists.